dist/minimongo-observable/to-observable.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 var observable_cursor_1 = require('./observable-cursor');
+var _ = require('lodash');
+var COLLECTION_EVENTS_DEBOUNCE_TIMEFRAME = 16;
 function toObservable(cursor) {
     var observable = observable_cursor_1.ObservableCursor.create(function (observer) {
```

Comment:
Trying to get the `Zone.current` - but all I get is the root zone, and when trying to execute `run` on it, the view does not update... any idea? @barbatus